diff --git a/src/store/modules/network/network.ts b/src/store/modules/network/network.ts
index 9d43c38..c1385a4 100644
--- a/src/store/modules/network/network.ts
+++ b/src/store/modules/network/network.ts
@@ -188,6 +188,15 @@ const network_module: Module<NetworkState, RootState> = {
                 true
             )
 
+            const avado = new AvaNetwork(
+                'Mainnet (AVADO)',
+                'https://avalanchego.my.ava.do:9650',
+                1,
+                'https://explorerapi.avax.network',
+                'https://explorer.avax.network',
+                true
+            )
+
             // Load custom networks if any
             try {
                 await dispatch('load')
@@ -195,6 +204,7 @@ const network_module: Module<NetworkState, RootState> = {
                 console.error(e)
             }
 
+            commit('addNetwork', avado)
             commit('addNetwork', mainnet)
             commit('addNetwork', fuji)
 
