diff --git a/src/store/modules/network/network.ts b/src/store/modules/network/network.ts
index df45ee0..7f76e36 100644
--- a/src/store/modules/network/network.ts
+++ b/src/store/modules/network/network.ts
@@ -164,7 +164,7 @@ const network_module: Module<NetworkState, RootState> = {
         },
 
         async init({ state, commit, dispatch }) {
-            // let netTest = new AvaNetwork("Everest TestNet", 'https://api.avax-test.network:443', 4, 'https://explorerapi.avax.network');
+            // let netTest = new AvaNetwork('Everest TestNet", 'https://api.avax-test.network:443', 4, 'https://explorerapi.avax.network');
             let manhattan = new AvaNetwork(
                 'Mainnet',
                 'https://api.avax.network:443',
@@ -189,6 +189,17 @@ const network_module: Module<NetworkState, RootState> = {
                 console.error(e)
             }
 
+            let avado = new AvaNetwork(
+                'Mainnet (AVADO)',
+                'https://avalanchego.avadopackage.com:9650',
+                1,
+                'https://explorerapi.avax.network',
+                'https://explorer.avax.network',
+                true
+            )
+
+            commit('addNetwork', avado)
+
             // commit('addNetwork', netTest);
             commit('addNetwork', manhattan)
             commit('addNetwork', fuji)
